cvat/settings/social_auth_settings.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2022 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
```

Comment:
I believe the file should be in tests directory. As I can see it is necessary only to run tests.